airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1158,22 +1156,29 @@ def __init__(
                 skip_leading_rows = 0
             if not field_delimiter:
                 field_delimiter = ","
+            if not destination_project_dataset_table:
+                raise ValueError(
+                    "`destination_project_dataset_table` is required when " "not using `table_resource`."
```

Comment:
```suggestion
                    "`destination_project_dataset_table` is required when not using `table_resource`."
```
Small string cleanup.